.github/workflows/publish-on-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish release to PyPI
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build-and-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the release commit
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install Python packages needed for build and upload
+      run: |
+        python -m pip install build twine
+
+    - name: Build sdist and wheel
+      run: |
+        python setup.py sdist
```

Comment:
Shouldn't this be `python -m build --sdist`? Or `python -m build` if you want an sdist and a wheel? And if you're building both, does the `twine` invocation need to change?

Reply:
Quite likely.  I don't yet have deep experience with the new packaging tools. `python -m build --sdist` seems to match my intent.

Reply:
I'm using [this](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html) as a reference here.